telegram/_giveaway.py

```diff
@@ -137,19 +137,14 @@ def __init__(
         self._freeze()
 
     @classmethod
-    def de_json(
-        cls, data: Optional[JSONDict], bot: Optional["Bot"] = None
-    ) -> Optional["Giveaway"]:
+    def de_json(cls, data: JSONDict, bot: Optional["Bot"] = None) -> "Giveaway":
         """See :meth:`telegram.TelegramObject.de_json`."""
         data = cls._parse_data(data)
 
-        if data is None:
-            return None
-
         # Get the local timezone from the bot if it has defaults
         loc_tzinfo = extract_tzinfo_from_defaults(bot)
 
-        data["chats"] = tuple(Chat.de_list(data.get("chats"), bot))
+        data["chats"] = tuple(de_list_optional(data.get("chats"), Chat, bot))
```

Comment:
```suggestion
        data["chats"] = de_list_optional(data.get("chats"), Chat, bot)
```
doesn't `de_list_optional` already return a tuple? Same for all other instances